api/app/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -116,3 +132,63 @@ export async function sandboxGetDocRefsAndUpsert({
 
   return result;
 }
+
+function addSandboxFields(docRef: DocumentReference): DocumentReference {
+  if (docRef.content) {
+    const fileExt = docRef.content[0]?.attachment?.contentType?.split("/")[1]?.trim();
```

Comment:
we have a function for this https://github.com/metriport/metriport/blob/6aed56ac95875148ed1ea4aa78794608144572de/api/app/src/external/commonwell/document/shared.ts#L31-L36

Reply:
Tried using it with both `attachment.title` and `attachment.contentType` - it returns an empty string for files of "tif" extension. See the image: https://imgur.com/f7PoIMF

Reply:
Resolved